test/src/com/xilinx/rapidwright/design/TestDesignTools.java

```diff
@@ -309,6 +311,11 @@ public void testGetTrimmablePIPsFromPinsBidir(boolean unrouteAll) {
                 "INT_X102Y429/INT.INODE_W_BLN_60_FT1->>IMUX_W2",                // E1 input
         });
 
+        for (PIP pip : net.getPIPs()) {
+            if (pip.toString().equals("INT_X102Y428/INT.INT_NODE_IMUX_50_INT_OUT0<<->>BYPASS_W14"))
+                pip.setIsReversed(true);
```

Comment:
I guess we are relying on the source of the PIPs to set this flag correctly for this change to work.  Will that always be true?

Reply:
It looks like that, if it came directly from a Vivado DCP, then this flag should be set correctly. In other cases (e.g. when a user creates a PIP) then it is incumbent on them to set the flag.

A future enhancement would provide a helper method that can analyze a list of PIPs and update such flags.

Reply:
I have filed #579 and #580 to help us track this issue and ensure this doesn't become a problem.